README.md

```diff
@@ -86,161 +86,204 @@ session-service$ export set SERVER_PORT=8090; mvn package -Dpackaging.type=jar &
 
 ### Create
```

Comment:
I noticed this in someone's code.  If you could point me to the generated documentation and the source sometime that would be good.